src/main/java/org/killbill/billing/overdue/api/OverdueCondition.java

```diff
@@ -35,4 +35,6 @@
     public PaymentResponse [] getResponseForLastFailedPaymentIn();
 
     public ControlTagType getControlTagType();
```

Comment:
Maybe in the configuration section instead (I'm more worried about people potentially having to change the XML rather than the Java API change).